releasenotes/1.9.0.md

```diff
@@ -0,0 +1,73 @@
+permalink: /releasenotes/1.9.0/
+---
+
+1. [Introduction](#intro)
+1. [Highlights](#highlights)
+1. [API Changes](#api)
+1. [Known Issues](#issues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [Apache jclouds 1.9.0 released](TODO). You can read the details of the specific JIRA issues addressed in this release at the [JIRA Release Notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12329006&styleName=Html&projectId=12314430).
+
+To get jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Highlights
+
+* Configured the Animal Sniffer plugin to make sure the code is runtime compatible with Java 6. This will help us support Android.
+* Added AWS Frankfurt region
+* Added AWS signature v4 (excluding S3)
+* Added C4 instance types to AWS EC2
+* Refactored the OkHttp driver to use the native API
+* Added a clean way to customize SSL/TLS connections using the OkHttp driver to avoid POODLE.
+* Initial support for Azure Compute
+* Added authentication to the Docker provider
+* Added the ProfitBricks API
+* Added the Shipyard API
+* Added San José and Honolulu regions to CloudSigma
+* Improved Google Compute Engine Compute support
+* Chef has been promoted to the main repo and updated to support Chef 12
+* Improved Google Compute Engine to use AutoValue
+* Support portable setting and getting container and object ACLs (public-read and private)
+* Support portable multi-object delete
+* Improved multi-part upload compatibility with InputStream Payload
+* Add support for user metadata with filesystem provider
+* Add multi-part upload support for generic S3
+
+
+## <a id="api"></a>API Changes
+
+* Downgraded Guava dependency to 16.0.1 but code will now support also versions 17 and 18.
+* Removed all async interfaces from all apis and providers
+* Do not provide AWS endpoint for generic S3
+* Require Azure API 2012-02-12 to allow support after [August 2015](http://blogs.msdn.com/b/windowsazurestorage/archive/2014/08/05/microsoft-azure-storage-service-version-removal.aspx)
+* Require Java 7 to use filesystem provider (enables user metadata support)
+
+TODO
+
+## <a id="issues"></a>Known Issues
+
+* Amazon S3 does not support signature v4 or Frankfurt
+* Filesystem provider does not support user metadata on Mac OS X due to [OpenJDK issue](https://bugs.openjdk.java.net/browse/JDK-8030048)
+
+TODO
+
+## <a id="credits"></a>Credits
+
+jclouds would like to thank everyone who contributed time and effort in order to make this release happen:
+
+* Apache jclouds [PMC](http://people.apache.org/committers-by-project.html#jclouds-pmc) and community for verifying the release.
+* Check out who has been busy on [Open Hub](https://www.openhub.net/p/jclouds/contributors?query=&sort=latest_commit).
+
+TODO: Special thanks to the contributors from Google and Microsoft for their contributions in GCP and Azure.
```

Comment:
Is the current sentence not enough? Perhaps link to the discussion threads on `dev@` announcing the contributions?